benches/Cargo.toml

```diff
@@ -29,6 +29,11 @@ path = "bitcoin/witness.rs"
 harness = false
 
 
+[[bench]]
+name = "aead"
```

Comment:
nit: should this have the name `chacha20poly1305` to match the type it's benching? Seems to be the pattern with the other benches.

Reply:
Yes that's a better name. I've changed it. Thanks.